xml/System.Composition.Hosting.Core/LifetimeContext.xml

```diff
@@ -241,9 +241,9 @@ A bound part instance is always protected by locking <see langword="this" />, an
         <param name="contract">The contract.</param>
         <param name="export">After this method returns, contains the contract instance if available; otherwise, <see langword="null" />.</param>
         <summary>Retrieves a contract instance from the composition context.</summary>
-        <returns>
-          <see langword="true" /> if the instance was retrieved; otherwise, <see langword="false" />.</returns>
+        <returns><see langword="true" /> if the instance was retrieved; otherwise, <see langword="false" />.</returns>
         <remarks></remarks>
+        <exception cref="T:System.Composition.Hosting.CompositionFailedException"></exception>
```

Comment:
Found how the exception can happen on a LifetimeContext.TryGetExport : https://github.com/dotnet/corefx/blob/master/src/System.Composition.Hosting/src/System/Composition/Hosting/Core/ExportDescriptorRegistry.cs#L47-L54

 

Where SR.CardinalityMismatch_TooManyExports is:
"Only one export for the contract '{0}' is allowed, multiple implementations were found."
 
Perhaps the description in the exception tag can be:
"Throws when multiple implementations for the contract are found rather than only one export"

Reply:
That should suffice. In case you're not familiar with the feature, GitHub allows adding code suggestions directly in a conversation, and the PR owner can merge them. Example:

```suggestion
        <exception cref="T:System.Composition.Hosting.CompositionFailedException">When multiple implementations for the contract are found rather than only one export.</exception>
```